dev/tools/roll_dev.dart

```diff
@@ -0,0 +1,150 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
```

Comment:
wdyt about creating `dev/tools/lib` and putting this there?  It'd allow things like Google's roll script to import this and run it inline rather than having to spawn a new Dart process.

Reply:
Done.